src/test/java/org/mockbukkit/mockbukkit/ServerMockTest.java

```diff
@@ -448,6 +450,13 @@ void clearRecipes_SomeRecipes_AllRecipesRemoved()
 		assertFalse(server.recipeIterator().hasNext());
 	}
 
+	@Test
+	void recipeIterator_HasDefaultCraftingRecipes()
+	{
+		Iterator<Recipe> actual = server.recipeIterator();
+		assertEquals(976, Iterators.size(actual));
```

Comment:
This test is not really feasible to have, there amount of recipes will change in the future with almost every release

Reply:
Thank you for your feedback Thorinwasher, I have updated the unit tests accordingly.